cves/CVE-2013-1896.yml

```diff
@@ -159,8 +188,12 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
+      There are multiple layers of security when it comes to http requests,
```

Comment:
This sounds like something that should go in `distrust input` rather than `defense in depth`

Reply:
I think it applies to both here, although it is true that you should not trust the input from an HTTP request, you should also provide multiple layers of security when validating an HTTP request so there are multiple walls to block potentially harmful input like what we see in this vulnerability.